#hardware parameters for robot arm hackathon

LINK_1_LENGTH : 0.12
LINK_2_LENGTH :  0.12
LINK_3_LENGTH : 0.055
LINK_3_Z_OFFSET: 0.015
J2_BASE_Z_OFFSET: 0.12
AXIS_1_COUNTS_PER_REV : 3600
AXIS_2_COUNTS_PER_REV : 3600
AXIS_3_COUNTS_PER_REV : 3600